soco/alarms.py

```diff
@@ -306,8 +306,11 @@ def get_alarms(zone=None):
             datetime.strptime(values['Duration'], "%H:%M:%S").time()
         instance.recurrence = values['Recurrence']
         instance.enabled = values['Enabled'] == '1'
-        instance.zone = [z for z in zone.all_zones
-                         if z.uid == values['RoomUUID']][0]
+        instance.zone = next((z for z in zone.all_zones
+                         if z.uid == values['RoomUUID']), None)
```

Comment:
PEP8 complains: This line should be indented justified with the first 'z' from the previous line.